src/System.Diagnostics.Tracing/src/ApiCompatBaseline.net462.txt

```diff
@@ -0,0 +1,7 @@
+# DevDiv: Bug 1179822: ProjectN is missing the implementation of some System.Diagnostics.Tracing contract API
```

Comment:
This is desktop, not ProjectN.  I think we need a targeting pack for Net 4.6.2 to get these.